spybot/remote/steam_api.py

```diff
@@ -13,6 +13,8 @@ def get_steam_users_playing_info(steam_ids: List[str]):
             steam_info.get("steamid", ""),
             steam_info.get("gameid", 0),
             steam_info.get("gameextrainfo", ""),
+            steam_info.get("avatar", ""),
+            steam_info.get("personastate", 0),
```

Comment:
I think at this point we should create a data class with named attributes and return this here instead of a tuple with five elements. Much cleaner.

Also the function return type should be explicitly specified, something like `List[SteamAccountInfo]`